packages/browser/src/autocapture-utils.ts

```diff
@@ -73,7 +73,11 @@ export function makeSafeText(s: string | null | undefined): string | null {
 export function getSafeText(el: Element): string {
     let elText = ''
 
-    if (shouldCaptureElement(el) && !isSensitiveElement(el) && el.childNodes && el.childNodes.length) {
+    if (
+        (isExplicitCapture(el) || (!isExplicitNoCapture(el) && !isSensitiveElement(el))) &&
```

Comment:
see note below too, but the checks we removed from `shouldCaptureElement` are covered more broadly inside `isSensitiveElement`